mantidproject/mantid
# New unscripted testing page in developer docs

This is the beginning of a new section in the developer docs to formally combine all of the unscripted testing instructions. These are also acceptance testing instructions and should be run through any time that a major change is being made to an area of the code.


**To test:**

Build the dev-docs-html target of MantidPlot:

- Is `Testing` listed in the main index
- On the testing index page should be
    - Muon Testing
    - Error Reporter Testing
    -  Project Recovery Testing
- In Muon testing there should be 4 separate tests
- In Error Reporter there should be 6 tests
- In Project Recovery there should be 6 tests

Does this update require release notes?
<!-
If yes, edit the file docs/source/release/... 
-->

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
